common/appointment/cancel.ts

```diff
@@ -1,12 +1,13 @@
 import { lecture as Appointment, lecture_appointmenttype_enum as AppointmentType } from '@prisma/client';
 import { prisma } from '../prisma';
-import { getStudent, User, userForPupil, userForStudent } from '../user';
+import { getScreener, getStudent, User, userForPupil, userForStudent } from '../user';
 import * as Notification from '../notification';
 import { getLogger } from '../logger/logger';
 import { getAppointmentForNotification } from './util';
 import { deleteZoomMeeting } from '../zoom/scheduled-meeting';
 import { PrerequisiteError, RedundantError } from '../util/error';
 import { getNotificationContextForSubcourse } from '../courses/notifications';
+import { Student } from '../../graphql/generated';
```

Comment:
Please avoid importing ./graphql (the API layer) in ./common (c.f. "onion architecture"). You can use the student (lowercase :/) import from `@prisma/client`

Reply:
That seems to have been a false auto-inserted import done by vscode, it wasn't even used.
I removed it. :)